File: 백준/Bronze/28214. 크림빵/크림빵.py
n, k, p = map(int, input().split())
bread = list(map(int, input().split()))

res = 0
for i in range(n):
  non = 0
  for j in range(k):
    if bread[i*k+j] == 0:
      non += 1
  if non < p:
    res += 1

print(res)

File: 백준/Bronze/28214. 크림빵/크림빵.cc
#include <iostream>
#include <vector>
using namespace std;

int main() {
  int n, k, p;
  vector<int> bread;

  cin >> n >> k >> p;
  
  for(int i=0; i<n*k; i++) {
    int x;
    cin >> x;
    bread.push_back(x);
  }

  int res = 0;
  for(int i=0; i<n; i++) {
    int non = 0;
    for(int j=0; j<k; j++) {
      if (bread[i*k+j] == 0) non++;
    }
    if (non < p) res++;
  }
  cout << res << endl;

  return 0;
}